addons/account/models/account_tax.py

```diff
@@ -675,7 +675,7 @@ def _compute_factor(self):
     @api.depends('invoice_tax_id.company_id', 'refund_tax_id.company_id')
     def _compute_company(self):
         for record in self:
-            record.company_id = record.invoice_tax_id and record.invoice_tax_id.company_id.id or record.refund_tax_id.company_id.id
+            record.company_id = record.invoice_tax_id and record.invoice_tax_id.company_id.id or record.refund_tax_id and record.refund_tax_id.company_id.id or self.env.user.company_id.id
```

Comment:
```suggestion
            record.company_id = record.invoice_tax_id and record.invoice_tax_id.company_id.id or record.refund_tax_id and record.refund_tax_id.company_id.id or self.env.company.id
```

Reply:
Thx 👍🏼 I thought about that after pushing 😄 
I will fix that, in case there is no other objection by @william-andre or anyone at Odoo